cvmfs/catalog_mgr_impl.h

```diff
@@ -1,3 +1,13 @@
+/*
+ * catalog_mgr_impl.h
```

Comment:
Please move the commit forward to this PR.  The point of this is to not merge something that has dangling changes.  What gets merged should run correctly through test suite and cpplint and should be consistent and beyond remaining doubts, if humanly possible.